apis/policies/v1alpha3/clusteradmissionpolicy_types.go

```diff
@@ -23,6 +23,11 @@ import (
 
 // ClusterAdmissionPolicySpec defines the desired state of ClusterAdmissionPolicy
 type ClusterAdmissionPolicySpec struct {
+	// PolicyServer identifies an existing PolicyServer resource.
+	// +kubebuilder:default:=default
```

Comment:
Nice, we don't have to do anything fancy to have `default` being written into this field :partying_face: